compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_loops_for_vectorization.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --iree-amdaie-insert-loops-for-vectorization %s | FileCheck %s
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-amdaie-insert-loops-for-vectorization))" %s | FileCheck %s
```

Comment:
Wow. The recent update wouldn't let us use a standalone pass the way it was? :/

Reply:
Ya I think the stand alone way just looks concise but under the hood it is with the builtin.module pass manager. But now we need to be more explicit in mentioning that this pass is meant for a function pass manager.